### ISSUE TITLE
params in tracePlot not working

### ISSUE THREAD

**Issue by qureshlatif** [mikemeredith/mcmcOutput#1]
Hello. I am trying to use the mcmcOutput::tracePlot function to look at a subset of parameters. The 'params' argument doesn't seem to be working for me. Even when I specify the parameters I want, I still get plots for everything. Maybe I'm missing something.

**Comment by mikemeredith** [comment]
Hi Quresh, The `tracePlot` function doesn't have the `params` argument, that applies to `diagPlot` only. Guess it would not be too difficult to add `params` to those simpler functions, though they were intended as a way to get everything plotted compactly.

**Comment by qureshlatif** [comment]
Ah! OK, I got it now. The diagPlot function works great and serves the function I was looking for! The reason I got confused was that when I pulled up the help file associated with 'tracePlot', I see "params" and "RhatBad" listed as arguments, but I didn't notice and wasn't getting that the help file is actually for 'diagPlot' and that 'tracePlot' is a more based function upon which 'diagPlot' is built. I don't see any need to add those arguments to the base functions. I just need to learn how to read and interpret help files better. Thank you!    On Wednesday, August 17, 2022 at 06:43:51 PM MDT, Mike Meredith ***@***.***> wrote:  
 
 


Hi Quresh, The tracePlot function doesn't have the params argument, that applies to diagPlot only. Guess it would not be too difficult to add params to those simpler functions, though they were intended as a way to get everything plotted compactly.
